docs/operator/configuration_parameters.md

```diff
@@ -446,8 +456,21 @@ attributes.
     correctly handling pre-flight CORS requests.
 
 ## Rucio configuration table
+* The Configuration Table is automatically created when initialized, and populated with the defaults below 
```

Comment:
It is not really populated with the defaults. The defaults are just "used" but in principle the config table is created empty. I would remove this :-)